pyspeckit/spectrum/models/ammonia15n.py

```diff
@@ -0,0 +1,1092 @@
+    tex: float or None
+        Excitation temperature. Assumed LTE if unspecified (``None``) or if
+        tex>trot.  This is the excitation temperature for *all* of the modeled
+        lines, which means we are explicitly assuming T_ex is the same for all
+        lines.
+    ntot: float
+        Total log column density of 15-NH3.  Can be specified as a float in the
+        range 5-25
+    width: float
+        Line width (Gaussian sigma) in km/s
+    xoff_v: float
+        Line offset in km/s
+    fortho: float
+        Fraction of 15-NH3 molecules in ortho state.  Default assumes all para
+        (fortho=0).
+    tau: None or float
+        If tau (optical depth in the 1-1 line) is specified, ntot is NOT fit
+        but is set to a fixed value.  The optical depths of the other lines are
+        fixed relative to tau_oneone
+    fillingfraction: None or float
+        fillingfraction is an arbitrary scaling factor to apply to the model
+    return_tau: bool
+        Return a dictionary of the optical depths in each line instead of a
+        synthetic spectrum
+    return_tau_profile: bool
+        Return a dictionary of the optical depth profiles in each line, i.e.,
+        the optical depths that will be used in conjunction with T_ex to produce
+        the synthetic spectrum
+    return_components: bool
+        Return a list of arrays, one for each hyperfine component, instead of
+        just one array
+    background_tb : float
+        The background brightness temperature.  Defaults to TCMB.
+    ignore_neg_models: bool
+        Normally if background=TCMB and the model is negative, an exception
+        will be raised.  This parameter will simply skip that exception.  Use
+        with extreme caution: negative models (absorption spectra against the
+        CMB) are not physical!  You may want to allow this in some cases
+        because there can be numerical issues where the model goes negative
+        when it shouldn't.
+    verbose: bool
+        More messages
+    debug: bool
+        For debugging.
+
+    Returns
+    -------
+    spectrum: `numpy.ndarray`
+        Synthetic spectrum with same shape as ``xarr``
+    component_list: list
+        List of `numpy.ndarray`'s, one for each hyperfine component
+    tau_dict: dict
+        Dictionary of optical depth values for the various lines
+        (if ``return_tau`` is set)
+    """
+
+    from .ammonia15n_constants import (ckms, ccms, h, kb,
+                                    Jortho, Jpara, Brot, Crot)
+
+    # Convert X-units to frequency in GHz
+    if xarr.unit.to_string() != 'GHz':
+        xarr = xarr.as_unit('GHz')
+
+    if tex is None:
+        log.warning("Assuming tex=trot")
+        tex = trot
+    elif isinstance(tex, dict):
+        for k in tex:
+            assert k in line_names,"{0} not in line list".format(k)
+        line_names = tex.keys()
+    elif tex > trot:                                             									 #TO BE MODIFIED
```

Comment:
there's a lot of extra whitespace here, and the comment is unclear - can you clarify why this is to be modified?
```suggestion
    elif tex > trot:  #TO BE MODIFIED
```

Reply:
This comes from copy-paste, so both whitespaces and the comments themselves can be removed for me.